.github/workflows/bundle-size/package.json

```diff
@@ -4,8 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "build": "node build.js",
-    "preinstall": "node ./unlock-package-lock.cjs"
```

Comment:
This script had no effect, because `preinstall` actually runs after all modules already installed 🤷 
https://github.com/npm/cli/issues/2660